apps/morph_stats

```diff
@@ -123,8 +126,16 @@ def main(args):
 
 
 if __name__ == '__main__':
-    _args = parse_args()
+    _args = get_parser().parse_args()
     logging.basicConfig(level=(logging.WARNING,
                                logging.INFO,
                                logging.DEBUG)[min(_args.verbose, 2)])
+
+    if _args.list:
+        print nm.fst._get_doc()  # pylint: disable=W0212
```

Comment:
At some point the `get()` function should move to a public API module (together with other feature-getter helper code). This is a remnant of the section tree implementation work, which I didn't get to clean up completely.